tools/pythonpkg/tests/fast/pandas/test_pandas_arrow.py

```diff
@@ -0,0 +1,129 @@
+import duckdb
+import pytest
+import datetime
+
+from conftest import pandas_supports_arrow_backend
+pd = pytest.importorskip("pandas", '2.0.0')
+import numpy as np
+
+@pytest.mark.skipif(not pandas_supports_arrow_backend(), reason="pandas does not support the 'pyarrow' backend")
+class TestPandasArrow(object):
```

Comment:
Is it possible to create a pandas dataframe with arrow, numpy and object columns?

Reply:
That's done in `test_mixed_columns`
But I've added another test to it that should make this more explicit 👍